source/manual/post-a-statuspage-message.html.md

```diff
@@ -99,6 +99,20 @@ Example messages:
 <!-- -->
 > The invading alien army has retreated. Everything is fine again.
 
+## Posting a status update retrospectively ("Backfilling")
+
+Sometimes it may be necessary to post a status update after an incident has been resolved.
+
+1. Tick the `backfill` option
+1. Add the date of the incident and a suitable description. Bear in mind when writing the message that it will automatically be marked as resolved.
```

Comment:
1, 1, 1 should be 1, 2, 3 I assume?

Reply:
I've based it on https://github.com/alphagov/govuk-developer-docs/blob/main/source/manual/ruby.html.md?plain=1 - that seems to be how numbered lists are coded?